plugins/pulp_docker/plugins/importers/importer.py

```diff
@@ -265,14 +251,14 @@ def _import_manifests(conduit, units):
         # All manifests if not specified
 
         if units is None:
-            criteria = UnitAssociationCriteria(type_ids=[Manifest.TYPE_ID])
+            criteria = UnitAssociationCriteria(type_ids=[constants.MANIFEST_TYPE_ID])
             units = conduit.get_source_units(criteria=criteria)
```

Comment:
I don't think the conduits have been changed as part of the mongoengine conversion. My goal in this commit was just to replace the old-style json models with new-style ones and to fix whatever broke as a result. So I agree we should update this too, but like you suggested I'd rather do that later when the world isn't crashing down on Pulp.